WooCommerce/Classes/ViewRelated/Orders/Order Creation/ProductsSection/ProductInOrder.swift

```diff
@@ -33,13 +35,40 @@ struct ProductInOrder: View {
                                 .accessibilityIdentifier("add-discount-button")
                             Divider()
                         }
-                        .renderedIf(viewModel.isAddingDiscountToProductEnabled)
+                        .renderedIf(viewModel.showAddDiscountRow)
                     }
                     .background(Color(.listForeground(modal: false)))
                     .sheet(isPresented: $shouldShowDiscountLineDetails) {
                         FeeOrDiscountLineDetailsView(viewModel: viewModel.discountDetailsViewModel)
                     }
                     Spacer(minLength: Layout.sectionSpacing)
+
+                    Section {
+                        Divider()
+                        VStack(alignment: .leading, spacing: Layout.noSpacing) {
+                            HStack() {
+                                Text(Localization.discountTitle)
+                                    .headlineStyle()
+                                Spacer()
+                                Button(Localization.editDiscount) {
+                                    shouldShowDiscountLineDetails = true
+                                }
+                            }
+
+                            Spacer()
+                            Text(Localization.discountAmount)
+                                .subheadlineStyle()
+                            Text(viewModel.formattedDiscount ?? "")
```

Comment:
I think this causes to see a line with a `$0.00` discount when is removed from the Order due to the currency formatter applying a format when the string it's empty. Perhaps we can do something like:
```
if let formattedDiscount = viewModel.formattedDiscount {
	Text(formattedDiscount)
}
```

Reply:
Good hint, I changed it here `1ce8f19fd4963f0cf6fa5b313336aa85d36c57b1` Now we change if the formattedDiscount is not nil and render it accordingly.